controllers/operator_controller.go

```diff
@@ -145,11 +154,47 @@ func (r *OperatorReconciler) reconcile(ctx context.Context, op *operatorsv1alpha
 	return ctrl.Result{}, nil
 }
 
+func (r *OperatorReconciler) generateExpectedBundleDeployment(o operatorsv1alpha1.Operator, bundlePath string) (*rukpakv1alpha1.BundleDeployment, error) {
+	dep := &rukpakv1alpha1.BundleDeployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: o.GetName(),
+		},
+		Spec: rukpakv1alpha1.BundleDeploymentSpec{
+			//TODO: Don't assume plain provisioner
```

Comment:
+1 

Long term, I envision a separate configuration API that sets up mappings from bundle properties to provisioner class names.

Mid term, perhaps we get some mileage around a property that sets the provisioner class name directly. We have to be careful with this though because I expect `ProvisionerClass` could end up being a new API in rukpak, which means provisionerClassName becomes a runtime thing, not something hardcoded in rukpak.

Short term, what you have here will be fine for quite awhile, I imagine.